src/js/player.js

```diff
@@ -4133,6 +4133,7 @@ class Player extends Component {
     }
 
     this.language_ = String(code).toLowerCase();
+    this.trigger('languagechange');
```

Comment:
If the same language was set, we aren't actually changing the language and we shouldn't trigger the event.